GrandCentralBoard/Widgets/GitHub/View/GitHubWidget.swift

```diff
@@ -7,22 +7,37 @@
 
 import GCBCore
 
+private let widgetTitle = "GitHub".uppercaseString
```

Comment:
We had another PRs like that, and decided to use camel case and `uppercaseString` for readability. For example here: https://github.com/macoscope/GrandCentralBoard/pull/106#discussion_r62490755

So I am just following previous decision and current code style.